gensim/models/word2vec_inner.pyx

```diff
@@ -71,10 +71,10 @@ cdef void our_saxpy_noblas(const int *N, const float *alpha, const float *X, con
     for i from 0 <= i < N[0] by 1:
         Y[i * (incY[0])] = (alpha[0]) * X[i * (incX[0])] + Y[i * (incY[0])]
 
-cdef long long _mul(const np.uint32_t a, const int b) nogil: 
+cdef long long _mul(const np.uint32_t a, const int b) nogil:
     """Safe multiplication of ints with explict typecasting"""
     return <long long>a * <long long>b
-        
```

Comment:
Nice catch! Trailing whitespace should never have been merged. CC @mpenkov .

Reply:
To be fair, the text editor did this!